tests/BC003_AL01/readme.md

```diff
@@ -0,0 +1,124 @@
+| NE01      | Network integrity/continuity check on linear elements |
+| ALX4      | Stationing tests on cable joints (punctual) |
+| NE02      | Cable routing |
+
+
+## Prerequisites
+
+None.
+
+## Test dataset (input)
+
+The dataset is composed of a LandXML file that contains alignments for a section of a 1.7 kilometers long tramway line. In this file you’ll find 4 aligments that compose this section.
+The coordinates system is based on RGF93 Lambert Zone 3 ([EPSG 3944](https://epsg.io/3944)) and the vertical datum NGF IGN69 ([EPSG:5720](https://epsg.io/5720)).
+
+This test case utilises the dataset collected in the Dataset folder and is summarised in the table below. **For more details on each item see [Dataset description](Dataset/README.md).**
+
+| Filename (format)         | Description                                                        |
+|---------------------------|--------------------------------------------------------------------|
+| BC003_AL01_reference.ifc  | **Reference IFC file.** Contains an exemplary export for this test. Please, note that this IFC file was created using the existing capabilities of Civil 3D, which means that the file is not 100% compliant with the test requirements.|
+| BC003_AL01_alignments.dwg  | **Reference native file.** It’s an Autodesk Civil 3D 2023 file containing the 4 project alignments.|
+| BC003_AL01_alignments.xml  | **The exported LandXML** format from the native file containing the same 4 alignments.|
+
+In addition, the outcome of the test PS01 shall be used as input of this test.
+
+## Validation criteria
+
+:zap: For this test case to be considered passed, **all criteria listed in this section**, and **the ones of prerequisites tests** shall be verified. :zap:
+
+### Formal rules
+
+#### IFC standard (schema and specification)
+
+When validated using the bSI Validation Service, the IFC must pass:
+
+- Syntax & Schema check
+
+
+#### Test case-specific checks
+
+[IDS file BC003_AL01.ids](./Dataset/BC003_AL01.ids):
+
+- There must be 1 instance of `IfcProject` and must be named `BC003: Test plan 1` and the `GUID “3cyAkba2v5a9pVuthidcpX”`.
+- Each `IfcAlignment` shall have a Name which is the same with the one in the LandXML file.
+- Each `IfcAlignment` must have a start station.
+- The horizontal layout must include only the following types of segments: Line, Circular Arc, Clothoid
+- The vertical layout must include only the following types of segments: Constant Gradient, Parabolic Arc
+
+Not covered by IDS but information can be extracted from it for external analysis:
+- There must be 4 instance(s) of `IfcAlignment`.
+
+Not covered by IDS file (must be checked otherwise):
+- Each `IfcAlignment` must have OwnerHistory assigned, which has OwningUser, OwningApplication, ChangeAction, CreationDate and State. The `IfcOwnerHistory` can be shared by multiple `IfcAlignment`(s) if they have the same information.
+- (TBD) Each `IfcAlignment` must be aggregated directly under `IfcProject`.
```

Comment:
This is the same as [ALB001](https://github.com/buildingSMART/ifc-gherkin-rules/blob/main/features/ALB001_Alignment-in-spatial-structure.feature) that was waiting for IF agreement to be modified accordingly...and later became (why?) [ALB004](https://github.com/buildingSMART/ifc-gherkin-rules/pull/67). Either cases (which we need to resolve), this shall be moved in the IFC standard part.